JourneysSquad/JourneysSquad/Model/DummyBillModel.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// A statement of the money owed for goods or services by different people.
+struct DummyBillModel {
+
+    /// The description of the bill.
+    let description = "Description"
+
+    /// Participants who is on the account.
```

Comment:
```suggestion
    /// Participants that paid within the bill.
```
You have such notation for property below.

Reply:
Documentation has been changed by semantic meaning.